### ISSUE TITLE
Starting service is too slow

### ISSUE THREAD

**Issue by Kevin-free** [Shaunwei/RealChar#325]
Is there a way to quickly restart the service? For example, just modify the configuration and start hot directly.

<img width="907" alt="image" src="https://github.com/Shaunwei/RealChar/assets/26422757/b369f2aa-0f12-4070-8340-758bf571461a">

**Comment by pycui** [comment]
yes you can set `OVWRITE_CHROMA=false` in your .env (we just put it in the .env.example), it will only take a few secs to run.
You can then use the `--reload` feature of uvicorn so everytime you make some change it will reload automatically

**Comment by Kevin-free** [comment]
> yes you can set `OVERWRITE_CHROMA=false` in your .env (we just put it in the .env.example), it will only take a few secs to run. You can then use the `--reload` feature of uvicorn so everytime you make some change it will reload automatically

Thanks!
